### PR TITLE
feat(FEC-12859) | Exposed API to handle lens/glass size for VR Medias

### PR DESCRIPTION
**Pull request by GouravSna**
- Added Distortion Config on `VRSettings`.

```kotlin
vrSettings.vrDistortionConfig = VRDistortionConfig().apply {
            scale = 0.75f
        }

```
Other params which can be used,

```java
data class VRDistortionConfig(var paramA: Double = DEFAULT_PARAM_A,
                              var paramB: Double = DEFAULT_PARAM_B,
                              var paramC: Double = DEFAULT_PARAM_C,
                              var scale: Float = DEFAULT_BARREL_DISTORTION_SCALE,
                              var defaultEnabled: Boolean = false)
```

### CODE REVIEW

File: playkit/src/main/java/com/kaltura/playkit/player/vr/VRBarrelDistortionConfig.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.kaltura.playkit.player.vr
+
+data class VRBarrelDistortionConfig(var paramA: Double = -0.068, // affects only the outermost pixels of the image
```
Comment (giladna) [inline playkit/src/main/java/com/kaltura/playkit/player/vr/VRBarrelDistortionConfig.kt]: @GouravSna  lets give meaning full name for the outside wold maybe?

File: playkit/src/main/java/com/kaltura/playkit/player/vr/VRBarrelDistortionConfig.kt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                                    var paramB: Double = 0.320000, // most cases only require b optimization
+                                    var paramC: Double = -0.2, // most uniform correction
+                                    private var _scale: Float = DEFAULT_BARREL_DISTORTION_SCALE,
```
Comment (giladna) [inline playkit/src/main/java/com/kaltura/playkit/player/vr/VRBarrelDistortionConfig.kt]: add comment for valid range?

File: playkit/src/main/java/com/kaltura/playkit/player/vr/VRBarrelDistortionConfig.kt (hunk 1 above)
Comment (GouravSna) [reply playkit/src/main/java/com/kaltura/playkit/player/vr/VRBarrelDistortionConfig.kt]: Done.

File: playkit/src/main/java/com/kaltura/playkit/player/vr/VRBarrelDistortionConfig.kt (hunk 2 above)
Comment (GouravSna) [reply playkit/src/main/java/com/kaltura/playkit/player/vr/VRBarrelDistortionConfig.kt]: Done.

**Review by giladna — APPROVED** [review]

**Pull request merged**